### ISSUE TITLE
Remove options -c 4 from SYSLOGD_OPTIONS in RHEL7

### ISSUE THREAD

**Issue by gothicx** [saz/puppet-rsyslog#206]
In the manual of rsyslog on RHEL 7.2 it says:
 -c version
              This option has been obsoleted and has no function any longer. It is still accepted in order not to break existing scripts. **However, future versions may not support it**.

Basically it should not deploy any SYSLOGD_OPTIONS options in /etc/sysconfig/rsyslog for RHEL7.

Default file:
```
# Options for rsyslogd
# Syslogd options are deprecated since rsyslog v3.
# If you want to use them, switch to compatibility mode 2 by "-c 2"
# See rsyslogd(8) for more details
SYSLOGD_OPTIONS=""
```

**Comment by saz** [comment]
This should be resolved with release 4.0.0

**Comment by gothicx** [comment]
Thank you Steffen.
